cps/templates/feed.xml

```diff
@@ -49,26 +49,32 @@
       </author>
     {% endif %}
     {% if entry.publishers.__len__() > 0 %}
-      <publisher>
+      <dc:publisher>
         <name>{{entry.publishers[0].name}}</name>
-      </publisher>
+      </dc:publisher>
```

Comment:
According to specification correct, calibre doesn't output the entry
-> better not changing unless really needed